### PR TITLE
feat: Download questionnaire responses

### LINKED ISSUE #4981: Download questionnaire responses
### Description

The questionnaire responses, filed in the feedback section, are often sent to the confirming body. It would be convenient to have a "download to file" button. This would address 8bit chars in content, too.

### Code of Conduct

- [X] I agree to follow the [IETF's Code of Conduct](https://github.com/ietf-tools/.github/blob/main/CODE_OF_CONDUCT.md)

### PR DESCRIPTION
**Pull request by pselkirk**
Fixes #4981

### CODE REVIEW

File: ietf/templates/nomcom/download_questio.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{# Copyright The IETF Trust 2023, All Rights Reserved #}{% autoescape off %}{% load nomcom_tags %}Questionnaire response from {{ nominee.person.name }}
+
+From:      {{ feedback.author|formatted_email|default:"Anonymous" }}
+Date:      {{ feedback.time|date:"Y-m-d" }}
+Positions: {{ positions }}{% if feedback.subject %}
```
Comment (pselkirk) [inline ietf/templates/nomcom/download_questio.txt:5]: For some reason, `{{ feedback.positions.all|join:"," }} ` renders fine in the html template, but renders as `<PositionQuerySet [<Position: Test Area Director>]>` here, so I moved the join to the view.

File: ietf/nomcom/views.py [hunk 2]
```diff
@@ -1023,24 +1024,38 @@ def view_feedback_nominee(request, year, nominee_id):
...
         feedback = get_object_or_404(Feedback, id=feedback_id)
-        type = request.POST.get('type', None)
-        if type:
-            if type == 'unclassified':
-                feedback.type = None
-                feedback.nominees.clear()
-                messages.success(request, 'The selected feedback has been de-classified. Please reclassify it in the Pending emails tab.')
+        submit = request.POST.get('submit', None)
+        if submit == 'download':
+            fn = f'questio-{slugify(nominee.name())}-{feedback.time.date()}.txt'
```
Comment (rjsparks) [inline ietf/nomcom/views.py:1029]: Leaking the slug out here isn't necessary (it's unfortunate that the slug is truncated to start with - holdover from a much older version of django). Please change this to be questionnaire-

File: ietf/templates/nomcom/download_questio.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{# Copyright The IETF Trust 2023, All Rights Reserved #}{% autoescape off %}{% load nomcom_tags %}Questionnaire response from {{ nominee.person.name }}
```
Comment (rjsparks) [inline ietf/templates/nomcom/download_questio.txt:1]: Similarly, this filename is jarring - please rename it download_questionnaire.txt

File: ietf/templates/nomcom/download_questio.txt (hunk 1 above)
Comment (rjsparks) [reply ietf/templates/nomcom/download_questio.txt:5]: Curious, likely to be a consequence of render_to_string, but not worth pursuing further at this time.

**Review by rjsparks — CHANGES_REQUESTED** [review]
A couple of nits to address please

**Review by rjsparks — APPROVED** [review]

**Pull request merged**